source/puppet-agent/_release_notes.md

```diff
@@ -1,10 +1,13 @@
 ### Release Notes
 
+* **1.2.7**, Oct. 29, 2015: Updates Ruby, [Facter](/facter/3.1/release_notes.html#facter-311), and [Hiera](/hiera/latest/release_notes.html#hiera-304). This version also [moves the OS X bill of materials](https://tickets.puppetlabs.com/browse/PA-21) to `/usr/local/share` on all supported OS X platforms to ensure future compatibility with OS X 10.11 (El Capitan), and [adds a root certificate](https://tickets.puppetlabs.com/browse/PA-20) in Enterprise Linux 4 to enable Forge access.
```

Comment:
I think we only bumped Ruby to 1.0.2d for Windows, as the other platforms were already there.

Also could you add a note that puppet was updated too?